<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>501</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>501</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>501</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>GF2 Function Generator</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/icons/icons/active64.png</normaloff>
    <normalon>:/icons/icons/active64.png</normalon>
    <disabledoff>:/icons/icons/greyed64.png</disabledoff>
    <disabledon>:/icons/icons/greyed64.png</disabledon>
    <activeoff>:/icons/icons/active64.png</activeoff>
    <activeon>:/icons/icons/active64.png</activeon>
    <selectedoff>:/icons/icons/selected64.png</selectedoff>
    <selectedon>:/icons/icons/selected64.png</selectedon>:/icons/icons/active64.png</iconset>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="groupBoxGenParams">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>5</y>
      <width>381</width>
      <height>391</height>
     </rect>
    </property>
    <property name="title">
     <string>Generation Parameters</string>
    </property>
    <widget class="QLabel" name="labelWave">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>37</y>
       <width>115</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Waveform:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="labelFreq0">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>99</y>
       <width>115</width>
       <height>33</height>
      </rect>
     </property>
     <property name="text">
      <string>Frequency 0 (KHz):</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="labelFreq1">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>139</y>
       <width>115</width>
       <height>33</height>
      </rect>
     </property>
     <property name="text">
      <string>Frequency 1 (KHz):</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="labelPhase0">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>227</y>
       <width>115</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Phase 0 (°):</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="labelPhase1">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>267</y>
       <width>115</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Phase 1 (°):</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="labelAmp">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>315</y>
       <width>115</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Amplitude (V&lt;sub&gt;pp&lt;/sub&gt;):</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBoxWave">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>30</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>Sinusoidal</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Triangular</string>
      </property>
     </item>
    </widget>
    <widget class="QDoubleSpinBox" name="doubleSpinBoxFreq0">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>100</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="decimals">
      <number>4</number>
     </property>
     <property name="maximum">
      <double>40000.000000000000000</double>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="doubleSpinBoxFreq1">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>140</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="decimals">
      <number>4</number>
     </property>
     <property name="maximum">
      <double>40000.000000000000000</double>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="doubleSpinBoxPhase0">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>220</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="maximum">
      <double>360.000000000000000</double>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="doubleSpinBoxPhase1">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>260</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="maximum">
      <double>360.000000000000000</double>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="doubleSpinBoxAmp">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>310</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="decimals">
      <number>3</number>
     </property>
     <property name="maximum">
      <double>8.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.100000000000000</double>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBoxActiveFreq">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>70</y>
       <width>131</width>
       <height>111</height>
      </rect>
     </property>
     <property name="title">
      <string>Active Frequency</string>
     </property>
     <widget class="QRadioButton" name="radioButtonSelFreq0">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>35</y>
        <width>111</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Frequency 0</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButtonSelFreq1">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>75</y>
        <width>111</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Frequency 1</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBoxActivePhase">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>190</y>
       <width>131</width>
       <height>111</height>
      </rect>
     </property>
     <property name="title">
      <string>Active Phase</string>
     </property>
     <widget class="QRadioButton" name="radioButtonSelPhase0">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>35</y>
        <width>111</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Phase 0</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButtonSelPhase1">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>75</y>
        <width>111</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Phase 1</string>
      </property>
     </widget>
    </widget>
    <widget class="QCheckBox" name="checkBoxZload">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>315</y>
       <width>131</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>50Ω load</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonApply">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>350</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Apply</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonClear">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>350</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Clear</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBoxControl">
    <property name="geometry">
     <rect>
      <x>395</x>
      <y>5</y>
      <width>101</width>
      <height>391</height>
     </rect>
    </property>
    <property name="title">
     <string>Control</string>
    </property>
    <widget class="QPushButton" name="pushButtonStart">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>81</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Start</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonStop">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>81</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Stop</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonReset">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>350</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Reset</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBoxDAC">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>130</y>
       <width>81</width>
       <height>101</height>
      </rect>
     </property>
     <property name="title">
      <string>DAC</string>
     </property>
     <widget class="QRadioButton" name="radioButtonDACOn">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>35</y>
        <width>123</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>On</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButtonDACOff">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>65</y>
        <width>123</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Off</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBoxClk">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>240</y>
       <width>81</width>
       <height>101</height>
      </rect>
     </property>
     <property name="title">
      <string>Clock</string>
     </property>
     <widget class="QRadioButton" name="radioButtonClkOn">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>35</y>
        <width>123</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>On</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButtonClkOff">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>65</y>
        <width>123</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Off</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QTextBrowser" name="textBrowserCommand">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>405</y>
      <width>491</width>
      <height>161</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Monospace</family>
      <pointsize>9</pointsize>
     </font>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>501</width>
     <height>29</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionLoad"/>
    <addaction name="actionSave"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuOptions">
    <property name="title">
     <string>&amp;Options</string>
    </property>
    <addaction name="actionAutoApply"/>
    <addaction name="actionResetClear"/>
    <addaction name="separator"/>
    <addaction name="actionSerial"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuOptions"/>
   <addaction name="menuHelp"/>
  </widget>
  <action name="actionQuit">
   <property name="text">
    <string>&amp;Quit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionAutoApply">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Auto Apply Changes</string>
   </property>
  </action>
  <action name="actionResetClear">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Clear after Reset</string>
   </property>
  </action>
  <action name="actionSerial">
   <property name="text">
    <string>&amp;Insert Serial Number</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>&amp;About GF2 GUI</string>
   </property>
  </action>
  <action name="actionLoad">
   <property name="text">
    <string>&amp;Load Settings</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>&amp;Save Settings</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionQuit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>250</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
